exodus_core/analysis/static_analysis.py

```diff
@@ -309,15 +309,23 @@ def _get_icon_from_gplay(self, handle, path):
         from bs4 import BeautifulSoup
         import urllib.request
 
+
         address = 'https://play.google.com/store/apps/details?id=%s' % handle
         text = urllib.request.urlopen(address).read()
         soup = BeautifulSoup(text, 'html.parser')
-        i = soup.find_all('img', {'class': 'cover-image', 'alt': 'Cover art'})
+        i = soup.find_all('img', {'class': 'T75of ujDFqe'})
         if len(i) > 0:
             url = '%s' % i[0]['src']
             if not url.startswith('http'):
                 url = 'https:%s' % url
-            f = urllib.request.urlopen(url)
+            short_url = ""
```

Comment:
This whole thing can just be replaced by the following line:
`short_url = url.partition("=")[0]`
or this one if you don't want to use another variable:
`url = url.partition("=")[0]`

Reply:
I agree.